Issue 1 / Update elm-css dependency to latest version

maxekman opened the pull request:
Allows the module to be used with the latest version of elm-css (6.1.0).

Also adds a Stylesheet.elm file for building a css file (for dev), adds content to the readme file and removes the dependency on scottcorgan/elm-html-template which was only used in the example and not when included as a dependency.

maxekman committed:
Bump the max version of elm-css to support 6.1.0, fixes #1

maxekman committed:
Add Stylesheets.elm file for generating CSS files with elm-css

maxekman committed:
Add usage and testing instructions

maxekman committed:
Remove dependency on scottcorgan/elm-html-template used in the example

maxekman commented:
By the way, I diffed the CSS file before and after the changes and it comes out clean.

scottcorgan commented on README.md:
```diff
@@ -1,3 +1,26 @@
+A port of normalize.css for elm-css.
+
+
+## Usage
```
I'd prefer Usage docs be restricted to examples in the Elm file's comments/docs. It's a typical Elm pattern to have a minimal README and put everything in the Elm files.

This is mostly so it shows up correctly on the Elm Package manager

maxekman replied:
Totally missed that there were docs already! I guess I'm just used to reading docs on Github. I'll add a link to the package docs instead then.

scottcorgan commented on README.md:
```diff
@@ -1,3 +1,26 @@
+Import the `Css.Normalize` module in your Stylesheets.elm file (or where you are building your elm-css) and include `Css.Normalize.css` in your list of CSS. See the examples folder for two uses; as inline styles and as .css file.
+
+
+## Development/Testing
```
👍 This is good to stay in the README

scottcorgan commented on README.md:
```diff
@@ -1,3 +1,26 @@
+
+`elm package install scottcorgan/elm-html-template && elm reactor` then navigate to examples/Main.elm
+
+Note that github.com/scottcorgan/elm-html-template is required for development/testing and has to be manually added.
```
💭 Just a thought. Maybe we add a `elm-package.json` file the the examples directory that includes the html template. Then there's less manual work when running `elm-reactor`.

scottcorgan reviewed (changes requested):
A few comments to make this a bit easier for everyone. Thanks for the hard work!

maxekman commented:
Thanks for the comments, will address them all soon. I did not know about the best practices for Elm package docs, thanks for pointing that out.

maxekman committed:
Remove dependencies for examples

maxekman committed:
Add separate elm-package.json file for the examples

maxekman committed:
Update readme to link to usage documentation instead of including it

maxekman commented:
I think I have addressed all comments in a sensible way. Don't forget to bump the version of this package in `elm-package.json` in the examples folder after bumping the real package version!

scottcorgan commented:
@maxekman this looks amazing! Thank you!

maxekman commented:
Awesome! Now I can use this package instead of a copy of the Normalize.elm file.

maxekman commented:
Worked like a charm to include it as a package with elm-css 6.1.0!
